### PR TITLE
feat: #422 add tags to statistics

### PR DESCRIPTION
**Pull request by Alex89198900**
1. Add tags to statistics instead of plain text

### CODE REVIEW

File: mw-webapp/src/logic/wayPage/wayStatistics/StatisticLine.tsx [hunk 1]
```diff
@@ -11,6 +11,11 @@ type StatisticLineProps = {
    * Line value (right part)
    */
   value: number | string;
+
+  /**
+   * Line component (right part)
+   */
+  component?: React.ReactNode;
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/wayStatistics/StatisticLine.tsx:18]: lets avoid to create additional props. Use polymorphism of description prop: 

```
descritpsion: string | React.ReactNode
```
code will be simplier  in this case

Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/wayStatistics/StatisticLine.tsx:18]: let's avoid creating additional props. Use polymorphism of description prop: 

```
description: string | React.ReactNode
```
code will be simpler  in this case

File: mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx [hunk 2]
```diff
@@ -22,11 +23,21 @@ export const TagStats = (props: TagStatsProps) => {
       <StatisticLine
         description={`${tagStat.name} jobs finished:`}
         value={`${tagStat.totalAmount} (${tagStat.totalAmountPercentage}%)`}
+        component={<JobTag
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx:26]: Mutually exclusive properties. Will disappear after unterface refactoring.

**Review by tritonJS826 — CHANGES_REQUESTED** [review]

**Review by Ekaterina1994 — APPROVED** [review]

File: mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx (hunk 2 above)
Comment (Alex89198900) [reply mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx]: Done

File: mw-webapp/src/logic/wayPage/wayStatistics/StatisticLine.tsx (hunk 1 above)
Comment (Alex89198900) [reply mw-webapp/src/logic/wayPage/wayStatistics/StatisticLine.tsx]: Done

File: mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx [hunk 3]
```diff
@@ -20,13 +21,23 @@ export const TagStats = (props: TagStatsProps) => {
   return props.stats.map((tagStat) => (
     <Fragment key={tagStat.name}>
       <StatisticLine
-        description={`${tagStat.name} jobs finished:`}
+        description={<JobTag
+          key={tagStat.name}
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx:25]: Why do we need key here?

File: mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx [hunk 4]
```diff
@@ -20,13 +21,23 @@ export const TagStats = (props: TagStatsProps) => {
...
+          isSmall
+        />}
         value={`${tagStat.totalAmount} (${tagStat.totalAmountPercentage}%)`}
+
       />
       <StatisticLine
         key={tagStat.name}
-        description={`${tagStat.name} time:`}
+        description={<JobTag
+          key={tagStat.name}
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx:35]: And here?)

Comment (Alex89198900) [reply mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx:35]: And here was the key before me

File: mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx (hunk 3 above)
Comment (tritonJS826) [reply mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx:25]: ok , check it

File: mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx (hunk 4 above)
Comment (tritonJS826) [reply mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx:35]: ok check it too

File: mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx (hunk 3 above)
Comment (Alex89198900) [reply mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx:25]: In this context, this is done as a warning from react: Each child in a list should have a unique "key" prop

File: mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx (hunk 4 above)
Comment (Alex89198900) [reply mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx:35]: In this context, this is done as a warning from react: Each child in a list should have a unique "key" prop

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**